sdk/core/test/integration/otp/index.test.ts

```diff
@@ -311,6 +312,25 @@ describe('CommonNetworkMember (flows that require OTP)', () => {
     expect(responseError.name).to.eql('COR-17')
   })
 
+  it('#passwordlessLogin with custom messages ', async () => {
+    const cognitoUsername = COGNITO_USERNAME
+    const stmp = `${Date.now()} stmp`
+    const messageParameters: MessageParameters = {
+      message: `Your verification code is ${stmp} {{CODE}}.`,
+      subject: `${stmp} {{CODE}}.`,
+    }
+    const fullOptions = getOptionsForEnvironment(true)
+
+    const token = await CommonNetworkMember.passwordlessLogin(cognitoUsername, fullOptions, messageParameters)
+
+    // NOTE: wait for 180s before providing the answer
+    await wait(DELAY)
+    const otp = await getOtp()
+    console.log(otp)
```

Comment:
@Volland please remove console.log, and add proper asserts to this test, cause its does not check nothing so far